packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -981,9 +980,9 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
           const abort = await this.registerDevice(FAILBACK_UTIL);
 
           if (!abort && !this.isDeviceRegistered()) {
-            const abort = await this.restorePreviousRegistration(FAILBACK_UTIL);
+            const abortNew = await this.restorePreviousRegistration(FAILBACK_UTIL);
 
-            if (abort) {
+            if (abortNew) {
```

Comment:
Can we rename this something better than abortNew? THe name of the variable doesn't really indicate what's different.